docs/src/how-to/install/ingress.md

```diff
@@ -38,8 +38,17 @@ To inspect default TLS settings, see [defaults in the latest code](https://githu
 
 ## Installing on bare-metal without dynamic load balancer support
 
-In case you cannot create a `kind: service` of `type: LoadBalancer`, then you
-can fall back to manually ensure traffic reaches your installation:
+```{note}
+Most managed K8s clusters have support for LoadBalancers. Kubernetes clusters that are manually set up can also support this, by using a provider/environment-specific [CCM (Cloud Controller Manager)](https://kubernetes.io/docs/concepts/architecture/cloud-controller/) (see hcloud and digitalocean for examples).
+
+In case you're provisioning on bare metal, without any hardware load balancer support in front,
+you might be using MetalLB, which supports BGP or Failover [L2 ARP announcements](https://metallb.universe.tf/configuration/_advanced_l2_configuration/).
```

Comment:
Looking at the ticket description, we probably shouldn't be linking to any of this repositories, but rather to https://kubernetes.github.io/ingress-nginx/deploy/baremetal/. We can still mention these as examples, but that should be it.

Reply:
Can you remove this paragraph describing MetalLB specifically? That's all included in https://kubernetes.github.io/ingress-nginx/deploy/baremetal/.